### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by archers1234**
update the readme to define what directory the watch is run in

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 ## BUILD INSTRUCTIONS
 * Run ```npm install``` in the ClientApp dir to install node dependancies
-* Run ```dotnet watch run``` to run app in watch mode. This starts up the server and client app, both will refresh on code changes.
+* Run ```dotnet watch run``` in the Server dir to run app in watch mode. This starts up the server and client app, both will refresh on code changes.
```
[the remark this answers is not in the document]
Comment (archers1234) [reply README.md]: do you need to run it in order to build it? is dotnet watch run part of the build proccess?

**Comment by ansh-projects** [comment]
No

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ansh-projects) [reply README.md]: There's quite a bit missing here lmao. You don't need dotnet run, you can just use dotnet build. However a run builds it. 

Dotnet publish builds both the server and client app in production settings.

[the remark this answers is not in the document]
Comment (archers1234) [reply README.md]: I am going to close this and then make any changes to it when i make the wiki stuff when i get time

**Pull request closed without merging**